metrics/grpc_mw_client.go

```diff
@@ -0,0 +1,74 @@
+package metrics
+
+import (
+	"context"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+var metricsCollector = make(map[string]any)
+
+// ClientMetricsUnaryInterceptor returns a new unary client interceptor for
+// metrics from what to
+// namespace here is a prefix for metrics name
+//
+// For every RPC it exports the following metrics:
+// - client_grpc_request_count{method, code,from,to}
+// - client_grpc_response_time{method,from,to}
```

Comment:
В одном месте после "method," стоит пробел, в другом нет. Можно сделать везде одинаково, чтобы сделать покрасивее 
```suggestion
// - client_grpc_request_count{method, code, from, to}
// - client_grpc_response_time{method, from, to}
```